modules/core/include/opencv2/core.hpp

```diff
@@ -862,8 +862,8 @@ CV_EXPORTS void minMaxLoc(const SparseMat& a, double* minVal,
 The function cv::reduce reduces the matrix to a vector by treating the matrix rows/columns as a set of
 1D vectors and performing the specified operation on the vectors until a single row/column is
 obtained. For example, the function can be used to compute horizontal and vertical projections of a
-raster image. In case of REDUCE_MAX and REDUCE_MIN , the output image should have the same type as the source one.
-In case of REDUCE_SUM and REDUCE_AVG , the output may have a larger element bit-depth to preserve accuracy.
+raster image. In case of cv::REDUCE_MAX and cv::REDUCE_MIN , the output image should have the same type as the source one.
+In case of cv::REDUCE_SUM and cv::REDUCE_AVG , the output may have a larger element bit-depth to preserve accuracy.
```

Comment:
if you want, you can replace "cv::" prefix here to "#" symbol ("%" for classes): https://www.stack.nl/~dimitri/doxygen/manual/autolink.html